mailviews/tests/emails/previews.py

```diff
@@ -1,7 +1,6 @@
 import random
 
 from django import forms
-from django.contrib.webdesign.lorem_ipsum import paragraphs, words
```

Comment:
This appears to have actually been moved to `django.utils.lorem_ipsum`, you can either catch the `ImportError` or just commit that change.

So now we don't have to pull from Robot Ipsum, and it doesn't change as much.